assets/prototype/domain/shared/dateRangeInput/formatSelect.tsx

```diff
@@ -0,0 +1,17 @@
+import moment from 'moment';
+
+interface MomentFormatter {
+	formatDate: (date: Date) => string;
+	parseDate: (str: string) => Date;
+	placeholder: string;
+}
+
+const momentFormatter = (format: string): MomentFormatter => ({
+	formatDate: (date: Date): string => moment(date).format(format),
+	parseDate: (str: string): Date => moment(str, format).toDate(),
+	placeholder: `${format} (moment)`,
+});
+
+export const FORMATS = {
+	MYSQL: momentFormatter('YYYY-MM-DD HH:mm:ss'),
+};
```

Comment:
The original code was written before we decided to go with `date-fns`, not sure if we want or not to keep moment along.
This is outside of the scope of the current PR.

Reply:
Yes, it caused errors at in its consumer components.

Reply:
created #2255 to deal with that